### DISCUSSION TITLE
How did horizon forbidden west handle cloth?

### DISCUSSION THREAD

**Discussion by JailbreakPapa** [jrouwe/JoltPhysics#303] [Q&A] [answered]
This might be a dumb question, but how/what did horizon forbidden west use for cloth?

**Comment by jrouwe** [comment, marked as answer]
As far as I know, we used a [Verlet based solver](http://graphics.cs.cmu.edu/nsp/course/15-869/2006/papers/jakobsen.htm) solving particles and stick constraints and a limited set of colliders (capsules) around the character.

B.t.w. Cloth physics is something that I would like to integrate in Jolt in the future.
